connectors/concur/src/main/java/com/vmware/connectors/concur/ConcurController.java

```diff
@@ -118,15 +168,16 @@ public ConcurController(WebClient rest,
         // Replace the placeholder in concur request template with appropriate action and comment.
         final String concurRequestTemplate = getConcurRequestTemplate(reason, concurAction);
 
-        return getWorkFlowActionUrl(authHeader, reportID, baseUrl)
-                .flatMap(url -> rest.post()
-                    .uri(url)
-                    .header(AUTHORIZATION, authHeader)
-                    .contentType(APPLICATION_XML)
-                    .accept(APPLICATION_JSON)
-                    .syncBody(concurRequestTemplate)
-                    .retrieve()
-                    .bodyToMono(String.class));
+        return fetchOAuthToken(authHeader, clientId, clientSecret)
+                .flatMap(oauthToken -> getWorkFlowActionUrl(oauthToken, reportID, baseUrl)
```

Comment:
This isn't the oauthToken.  It's the header value.  Call it oauthHeader or something, so it isn't confusing.